tests/automation_framework/data/worker/workerregister_applicationTypeId_32bytes.json

```diff
@@ -0,0 +1,25 @@
+{
+    "jsonrpc": "2.0",
+    "method": "WorkerRegister",
+    "id": 1,
+    "params": {
+        "workerId":"",
+        "workerType": 1,
+        "organizationId":"0xAD",
+        "applicationTypeId": "",
```

Comment:
Why is the applicationTypeId set to empty here when the test case is for 32 bytes?